PhysicsTools/NanoAOD/python/electrons_cff.py

```diff
@@ -159,45 +159,48 @@ def _get_bitmapVIDForEle_docstring(modules,WorkingPoints):
     srcLep = cms.InputTag("slimmedElectrons"),
     srcVtx = cms.InputTag("offlineSlimmedPrimaryVertices"),
 )
-for modifier in run2_egamma_2017,run2_egamma_2018, run2_miniAOD_80XLegacy,run2_nanoAOD_94XMiniAODv1,run2_nanoAOD_94XMiniAODv2,run2_nanoAOD_94X2016,run2_nanoAOD_102Xv1,run2_nanoAOD_106Xv1:
+for modifier in run2_egamma_2016,run2_egamma_2017,run2_egamma_2018, run2_miniAOD_80XLegacy,run2_nanoAOD_94XMiniAODv1,run2_nanoAOD_94XMiniAODv2,run2_nanoAOD_94X2016,run2_nanoAOD_102Xv1,run2_nanoAOD_106Xv1:
     modifier.toModify(ptRatioRelForEle, srcLep = "slimmedElectronsUpdated")
 
 seedGainEle = cms.EDProducer("ElectronSeedGainProducer", src = cms.InputTag("slimmedElectrons"))
-for modifier in run2_egamma_2017,run2_egamma_2018, run2_miniAOD_80XLegacy,run2_nanoAOD_94XMiniAODv1,run2_nanoAOD_94XMiniAODv2,run2_nanoAOD_94X2016,run2_nanoAOD_102Xv1,run2_nanoAOD_106Xv1:
+for modifier in run2_egamma_2016,run2_egamma_2017,run2_egamma_2018, run2_miniAOD_80XLegacy,run2_nanoAOD_94XMiniAODv1,run2_nanoAOD_94XMiniAODv2,run2_nanoAOD_94X2016,run2_nanoAOD_102Xv1,run2_nanoAOD_106Xv1:
```

Comment:
you can simplify combining here at the end all these toModify(..)

modifier.toModify(bitmapVIDForEle, src = "slimmedElectronsUpdated")
modifier.toModify(bitmapVIDForEleSpring15, src = "slimmedElectronsUpdated")
modifier.toModify(bitmapVIDForEleSum16, src = "slimmedElectronsUpdated")
modifier.toModify(bitmapVIDForEleHEEP, src = "slimmedElectronsUpdated")
modifier.toModify(isoForEle, src = "slimmedElectronsUpdated")
modifier.toModify(ptRatioRelForEle, srcLep = "slimmedElectronsUpdated")
modifier.toModify(seedGainEle, src = "slimmedElectronsUpdated")

Reply:
@sroychow @jainshilpi 
one question in view of the run2_nanoAOD_106Xv2 (i.e. those nano that will come out of the reminiv1)

for now basically you override the miniAOD content redoing in the nano since you have all these run2_egamma_2016,run2_egamma_2017,run2_egamma_2018
but this is not ideal, i.e. the dxy problem was fixed already there.

Reply:
@mariadalfonso just to understand better your last comment - do you mean the sign of dxy is fixed in run2_nanoAOD_106Xv2? 
In that case, we will have to make several changes again to revert from "slimmedElectronsUpdated" to "slimmedElectrons" for  run2_nanoAOD_106Xv2. By re-running this sign fix over nanoAOD (in a miniAOD which already has this sign fixed) is not optimal but if its fine, can we revisit this in another PR? Probably this config needs a lot of cleaning up and re-arrangement. Thanks

Reply:
the dxy has been fixed with https://github.com/cms-sw/cmssw/pull/30892 so it entered the remini_UL

Reply:
@jainshilpi @mariadalfonso I have addressed this in my last commit. Please have a look. slimmedElectronsUpdated is not run for run2_nanoAOD_106Xv2 in electronSequence and the modules running in the electron sequence have slimmedElectrons as input.